### DISCUSSION TITLE
Group some settings for common use.

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#641] [Ideas]
It would be nice to be able to create actions and then pass them around to all developers either as a must-have or nice to have.   Some commands are a little difficult for others to get right on their own and if I have already made actions that work they should use the ones I create.  

At this time I can't use workspace sync'ing.  Our location does not allow connection to GITHUB to MS from within VS CODE, regardless of the benefit this would have for giving others my changes to the settings.

**Comment by worksofliam** [comment]
Yes, sharing actions for all developers through the IFS would be aweesome.
